Mathlib/Data/ENNReal/Operations.lean

```diff
@@ -536,6 +536,50 @@ theorem iInf_add_iInf (h : ∀ i j, ∃ k, f k + g k ≤ f i + g j) : iInf f + i
       le_iInf₂ fun a a' => let ⟨k, h⟩ := h a a'; iInf_le_of_le k h
     _ = iInf f + iInf g := by simp_rw [iInf_add, add_iInf]
 
+lemma add_biInf' {p : ι → Prop} (f : ι → ℝ≥0∞) :
+    a + ⨅ i, ⨅ _ : p i, f i = ⨅ i, ⨅ _ : p i, a + f i := by
+  simp only [iInf_subtype', add_iInf]
+
+lemma biInf_add' {p : ι → Prop} (f : ι → ℝ≥0∞) :
+    (⨅ i, ⨅ _ : p i, f i) + a = ⨅ i, ⨅ _ : p i, f i + a := by
+  simp only [add_comm, add_biInf']
+
+lemma add_biInf {ι : Type*} {s : Set ι} (f : ι → ℝ≥0∞) :
+    a + ⨅ i ∈ s, f i = ⨅ i ∈ s, a + f i := add_biInf' _
+
+lemma biInf_add {ι : Type*} {s : Set ι} (f : ι → ℝ≥0∞) :
+    (⨅ i ∈ s, f i) + a = ⨅ i ∈ s, f i + a := biInf_add' _
+
+lemma add_sInf {s : Set ℝ≥0∞}: a + sInf s = ⨅ b ∈ s, a + b := by
+  rw [sInf_eq_iInf, add_biInf]
+
```

Comment:
You didn't add `sInf_add` --- intentional?

Reply:
It already exists